modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/utils/DeployEnvironments.java

```diff
@@ -12,21 +12,20 @@
 import org.eclipse.xpanse.modules.models.enums.DeployVariableKind;
 import org.eclipse.xpanse.modules.models.resource.DeployVariable;
 import org.eclipse.xpanse.modules.models.resource.Flavor;
+import org.springframework.stereotype.Component;
 
 /**
  * Environment variables utils for deployment.
  */
+@Component

```

Comment:
I think no need to make `DeployEnvironments` a `Component`, The methods here are sort of helpers for environments variables, no need to create an object.

Reply:
@iskey I made it a component so that it is possible to mock and test it without too much efforts. 
And it is also a singleton (by default) so no overhead we have here.

Reply:
Ok, It makes sense, but I feel it is a bit strange always.